### PR TITLE
feat(documentation): Add collaborators rights and dashboard managment

### PR DESCRIPTION
**Pull request by baptmaury**
Explain rights of collaborators and how to manage them from dashboard

Note: I don't understand why i don't find my collaborator page in the search engine, is it linked to algolia ?

Fix https://github.com/Scalingo/documentation/issues/1799

### CODE REVIEW

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Collaborators
+modified_at: 2024-02-07 00:00:00
+tags: app collaborators
+---
+
+## Context
+
+Collaborators allow you to add other accounts to help you manage your application. Collaborators basically have all the features of an owner account. It is also useful if you want to [transfer ownership of an application](https://doc.scalingo.com/platform/app/lifecycle-management#transfer-ownership-of-an-application).
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:9]: I suggest to re-phrase this like so:

```suggestion
Collaborators is a feature allowing the owner of an app to invite other Scalingo accounts to team up and work jointly on the app. An app can have as many collaborators as needed.

An owner of an app can also [transfer the ownership of the app]({% post_url platform/app/2000-01-01-lifecycle-management %}#transfer-ownership-of-an-app) to a collaborator.

{% note %}
Delegating rights to collaborators should be taken seriously. Even if collaborators can't delete the application, they have enough rights to disrupt the activities or businesses depending on it.
{% endnote %}
```

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+{% note %}
+To delete a collaborator, in the collaborator table, simply click on the trash can corresponding to the collaborator's mail and validate in the pop-up.
+{% endnote %}
+
+### From the CLI
+
+There you will learn how to manage collaborators from the CLI: [manage collaborators of the application](https://doc.scalingo.com/platform/cli/features#manage-collaborators-of-the-application)
+
+{% note %}
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:53]: I suggest to remove this note from here.
I put it back in the very first lines 😃

Comment (baptmaury) [reply src/_posts/platform/app/2000-01-01-collaborators.md:60]: Like top of the file or before From the dashboard ?

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Access the DB Dashboard
+- Almost all user actions except those under
+
+### Collaborator can't do
+
+- Delete application
+- Rename application
+- Transfer application
+
+## How to manage collaborators
```
[the remark this answers is not in the document]
Comment (baptmaury) [reply src/_posts/platform/app/2000-01-01-collaborators.md]: I have updated my text following your recomandation

**Review by Frzk — CHANGES_REQUESTED** [review]
Can you please put the commands in a code block?
The syntax is:
````
```shell
scalingo --app my-app whatever...
```
````

Then we're good to go, thanks a lot again <3

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+Using the Command Line
+
+```shell
+scalingo --app my-app collaborators
+```
+
+### Adding a Collaborator
+
+Using the Dashboard
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:53]:
```suggestion
#### Using the Dashboard
```

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+Using the Command Line
+
+```shell
+scalingo --app my-app collaborators-add user@example.com
+```
+
+### Removing a Collaborator
+
+Using the Dashboard
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:72]:
```suggestion
#### Using the Dashboard
```

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+- Go to your application that needs a new collaborator
+- Go to the Settings tab
+- You will automatically be taken to the Collaborators menu
+- Click on the Add button
+- Add the required collaborator email
+- You will now see the collaborator in light grey in the table.
+- The collaborator will receive an email to join the collaboration (and subscribe to Scalingo if required).
+- Once done, you will see the collaborator in black.
+
+Using the Command Line
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:64]:
```suggestion
#### Using the Command Line
```

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+Using the Dashboard
+
+- Go to your application that needs a new collaborator
+- Go to the Settings tab
+- You will automatically be taken to the Collaborators menu
+- Click on the trash can corresponding to the collaborator's mail
+- Validate in the pop-up
+
+Using the Command Line
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:80]:
```suggestion
#### Using the Command Line
```

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+- Delete the application
+- Rename the application
+- Transfer the ownership of the application to another user
+
+## Managing Collaborators
+
+### Listing Collaborators
+
+Using the Dashboard
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:39]:
```suggestion
#### Using the Dashboard
```

File: src/_posts/platform/app/2000-01-01-collaborators.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+- Go to your application that needs a new collaborator
+- Go to the Settings tab
+- You will automatically be taken to the Collaborators menu
+
+Using the Command Line
```
Comment (Frzk) [inline src/_posts/platform/app/2000-01-01-collaborators.md:45]:
```suggestion
#### Using the Command Line
```

**Review by Frzk — APPROVED** [review]
I added some h4 titles.
Else LGTM!
Nice work :heart:

**Pull request merged**